### PR TITLE
Fix the wrong display of tensor values in ExportedProgram

### PR DESCRIPTION
**Pull request by yijie-yang**

### CODE REVIEW

File: src/server/package/src/model_explorer/pytorch_exported_program_adater_impl.py [hunk 1]
```diff
@@ -30,10 +31,11 @@
 
 class PytorchExportedProgramAdapterImpl:
 
-  def __init__(self, ep: torch.export.ExportedProgram):
+  def __init__(self, ep: torch.export.ExportedProgram, settings: Dict):
```
Comment (jinjingforever) [inline src/server/package/src/model_explorer/pytorch_exported_program_adater_impl.py:34]: Could you also add the `settings` parameter to the following api too? In its comment, mention that currently we only support 'const_element_count_limit'.

https://github.com/google-ai-edge/model-explorer/blob/main/src/server/package/src/model_explorer/config.py#L68

Comment (yijie-yang) [reply src/server/package/src/model_explorer/pytorch_exported_program_adater_impl.py:34]: thanks for catching it, Jing!

File: src/server/package/src/model_explorer/pytorch_exported_program_adater_impl.py [hunk 2]
```diff
@@ -177,7 +179,12 @@ def add_node_attrs(self, fx_node: torch.fx.node.Node, node: GraphNode):
       if tensor_spec:
         node.attrs.append(KeyValue(key='target', value=str(tensor_spec[0])))
         node.attrs.append(
-            KeyValue(key='__value', value=self.print_tensor(tensor_spec[1]))
+            KeyValue(
+                key='__value',
+                value=self.print_tensor(
+                    tensor_spec[1], self.settings['const_element_count_limit']
```
Comment (jinjingforever) [inline src/server/package/src/model_explorer/pytorch_exported_program_adater_impl.py:185]: I think we need to check the existence of `const_element_count_limit`. If it doesn't exist, we should use the default value (16)

**Comment by justinchuby** [comment]
In https://github.com/google-ai-edge/model-explorer/blob/1153a8fa7b8f0b101124cf39e3df25d7e1397d62/src/server/package/src/model_explorer/pytorch_exported_program_adater_impl.py#L161,

I think `size_limit > total_size` should have been `size_limit >= total_size` so that a tensor with the exact size at the limit is printed with the correct shapes.

File: src/server/package/src/model_explorer/pytorch_exported_program_adater_impl.py (hunk 2 above)
Comment (yijie-yang) [reply src/server/package/src/model_explorer/pytorch_exported_program_adater_impl.py:192]: Done!

File: src/server/package/src/model_explorer/pytorch_exported_program_adater_impl.py [hunk 3]
```diff
@@ -30,10 +31,18 @@
...
+  ):
     self.ep = ep
     self.gm = self.ep.graph_module
     self.inputs_map = self.get_inputs_map()
+    if settings is None:
+      # Assigns the default values
+      settings = {
+          'const_element_count_limit': 16,
+      }
+    self.settings = settings
```
Comment (justinchuby) [inline src/server/package/src/model_explorer/pytorch_exported_program_adater_impl.py:45]: Probably need 

```python
self.settings.setdefault('const_element_count_limit', 16)
```
?

Comment (yijie-yang) [reply src/server/package/src/model_explorer/pytorch_exported_program_adater_impl.py:38]: When `settings` is `None`, we will need to initialize a new `Dict` with default values for `settings`.

**Review by jinjingforever — APPROVED** [review]

**Pull request merged**